lib/ansible/module_utils/facts/hardware/openbsd.py

```diff
@@ -119,13 +112,22 @@ def get_memory_facts(self):
         return memory_facts
 
     def get_uptime_facts(self):
-        uptime_facts = {}
-        uptime_seconds = self.sysctl['kern.boottime']
+        # On openbsd, we need to call it with -n to get this value as an int.
+        sysctl_cmd = self.module.get_bin_path('sysctl')
```

Comment:
why not modify the generic sysctl function to allow for args?

Reply:
Because in this case it would fail to parse. Using `-n` like we need here just gives an int back, not a `k=v` pair, so we'd fail to split on `=`.

So we can make `get_sysctl()` take an optional `raw=True` that returns the raw result...but then it has a different return type depending on args which feels like bad design.